src/OrchardCore.Modules/OrchardCore.Users/ViewModels/EditUserPhoneNumberViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace OrchardCore.Users.ViewModels;
+
+public class EditUserPhoneNumberViewModel
+{
+    [Phone(ErrorMessage = "Invalid Phone Number.")]
```

Comment:
I did what I saw with the other fields, I am not sure what do you mean by that

Reply:
I don't know the reason this is a checkbox for email since email validation is done using a confirmation process. But having it there for the email, it does not mean it is a good idea to do the same for phone number. Please remove the checkbox for the phone number

Reply:
Ok, you think its a good idea to keep the checkbox there, but make it readonly (just as an indicator)

Reply:
The indicator is a good idea but not the checkbox. I would just add a label next to the phone number "verified" in green or "unverified" in black. This way the indicator is on the same line at the phone number.

In a separate PR, we can do the same thing for the email confirmation later on.

Reply:
@hishamco will error message in a annotation attribute get localized? I remember there was a PR around this. I typically create the errors in the driver instead.

Reply:
It should be localized AFAIK